gapic/packaging/dependencies.yaml

```diff
@@ -49,7 +49,11 @@ gax_version:
   ruby:
     lower: '0.8.0'
   java:
-    lower: '0.6.1'
+    lower: '1.2.0'
+
+gax_grpc_version:
```

Comment:
Yes, I have one PR pending that will use this. This should not break the existing code though. I can revert this for now for safety.